Api/AbstractApi.php

```diff
@@ -183,6 +183,15 @@ protected function patch($name, $url)
      */
     protected function delete($name, $url)
     {
-        return $this->addRoute('delete', $name, new Route(Request::METHOD_DELETE, $url, $this->client, $this->eventDispatcher, $this->globalParameters));
+        return $this->addRoute('delete', $name, new Route(Request::METHOD_DELETE, $url, $this->client, $this->eventDispatcher, $this->globalParameters, $this->getName()));
+    }
+
+    /**
+     * @return string
```

Comment:
A comment may be useful here to understand why this function exist (every phpdoc comment in bundle should have description)